README.md

```diff
@@ -1,2 +1,29 @@
 # action-deploy-clojars
```

Comment:
Is it worth mentioning in this doc that you'll need our setup-env or at least setup-clojure to use this?

Reply:
Or maybe just more generally that you'll need the clojure CLI on there

Reply:
https://github.com/yetanalytics/action-deploy-clojars/pull/1/commits/dfe293df0d5e4c671e9e1b1a115e128e3cf674ec